README.md

```diff
@@ -130,6 +130,12 @@ logging:
   # regardless of the configured logging level.
   sampled_levels:
     'trace/webrequest': 0.2
+  # Set up the limiter for logging. When the rate of logs is close to the limit,
+  # it starts to increase the probability of discarding the logs of same kind.
+  # For default parameters use rate_limit: true
+  rate_limit:
```

Comment:
Hm.. The `sampled_levels` is a bit different - there we want to log something not legged normally with some small rate, while here we do not want to log something that's normally logged with a crazy rate. So conceptually it's pretty different..